hack/make/build-deb

```diff
@@ -8,16 +8,15 @@ set -e
 
 	# TODO consider using frozen images for the dockercore/builder-deb tags
 
-	tilde='~' # ouch Bash 4.2 vs 4.3, you keel me
-	debVersion="${VERSION//-/$tilde}" # using \~ or '~' here works in 4.3, but not 4.2; just ~ causes $HOME to be inserted, hence the $tilde
+	debVersion="${VERSION%%-*}"
```

Comment:
This will break release candidate ordering.  These strings were carefully constructed to order appropriately so that upgrades occur properly:
```console
$ dpkg --compare-versions 1.5.0 gt 1.5.0~rc1 && echo true || echo false
true
$ dpkg --compare-versions 1.5.0~rc1 gt 1.5.0~git20150128.112847.17e840a && echo true || echo false
true
$ dpkg --compare-versions 1.5.0~git20150128.112847.17e840a gt 1.5.0~dev~git20150128.112847.17e840a && echo true || echo false
true
```

This code as-is will strip the `-rc` suffix entirely. :smile:

These are also designed specifically to match as closely as we can to what actual Debian package versions look like for snapshot releases of packages (which is why there's a mismatch between our `.deb` versions and our `.rpm` versions, and I think that's totally reasonable and acceptable, given that the target distribution for each is different).